assets/aml-benchmark/components/batch-inference-preparer/spec.yaml

```diff
@@ -47,14 +51,19 @@ outputs:
   formatted_data:
     type: mltable
     description: Path to the folder where the payload will be stored.
+  output_metadata:
```

Comment:
nit: Can this be renamed as ground_truth_metadata?

Reply:
changed